main/app.js

```diff
@@ -1,6 +1,69 @@
-String.prototype.hasVowels = function() {};
-String.prototype.toUpper = function() {};
-String.prototype.toLower = function() {};
-String.prototype.ucFirst = function() {};
+String.prototype.hasVowels = function() {
+    return /[aeiou]/.test(this);
+};
+String.prototype.toUpper = function() {
+    var converted = '';
+    for (var index = 0; index < this.length; index++) {
+        converted += String.fromCharCode(this.charCodeAt(index) & 223);
+    }
+    return converted;
+};
+String.prototype.toLower = function() {
+    var converted = '';
+    for (var index = 0; index < this.length; index++) {
+        converted += String.fromCharCode(this.charCodeAt(index) | 32);
+    }
+    return converted;
+};
+String.prototype.ucFirst = function() {
+    return /[a-zA-Z]/.exec(this)[0] + this.substring(1).toLower();
+};
 
-module.exports = String.prototype;
+String.prototype.isQuestion = function() {
+    return /\?$/.test(this);
+};
+String.prototype.words = function() {
+    var pattern = /[a-zA-Z]/g;
+    return this.match(pattern);
+};
+String.prototype.wordCount = function() {
+    return this.words().length;
+};
+String.prototype.toCurrency = function() {
+    var digitPattern = /^[0-9]+[.]?[0-9]{1,2}$/;
+    return digitPattern.test(this)
+        ? Number(this)
```

Comment:
I don't think a ternary operator is the best implementation here. It seems to limit readability and the way it spans multiple lines without comments is not really good looking.